toonz/sources/translations/german/colorfx.ts

```diff
@@ -0,0 +1,723 @@
+        <translation>Dicke</translation>
+    </message>
+</context>
+<context>
+    <name>TCrystallizeStrokeStyle</name>
+    <message>
+        <source>Tulle</source>
+        <translation>Tüll</translation>
+    </message>
+    <message>
+        <source>Crease</source>
+        <translation>Falte</translation>
+    </message>
+    <message>
+        <source>Opacity</source>
+        <translation>Opazität</translation>
+    </message>
+</context>
+<context>
+    <name>TDottedFillStyle</name>
+    <message>
+        <source>Polka Dots</source>
+        <translation>Gepunktet</translation>
+    </message>
+    <message>
+        <source>Dot Size</source>
+        <translation>Punktgröße</translation>
+    </message>
+    <message>
+        <source>Dot Distance</source>
+        <translation>Punktdistanz</translation>
+    </message>
+</context>
+<context>
+    <name>TDottedLineStrokeStyle</name>
+    <message>
+        <source>Vanishing</source>
+        <translation>Verdünnung</translation>
+    </message>
+    <message>
+        <source>Fade In</source>
+        <translation>Einblende</translation>
+    </message>
+    <message>
+        <source>Dash</source>
+        <translation>Gestrichelt</translation>
+    </message>
+    <message>
+        <source>Fade Out</source>
+        <translation>Ausblenden</translation>
+    </message>
+    <message>
+        <source>Gap</source>
+        <translation>Lücke</translation>
+    </message>
+</context>
+<context>
+    <name>TDualColorStrokeStyle2</name>
+    <message>
+        <source>Striped</source>
+        <translation>Gestreift</translation>
+    </message>
+    <message>
+        <source>Distance</source>
+        <translation>Distanz</translation>
+    </message>
+</context>
+<context>
+    <name>TFriezeStrokeStyle2</name>
+    <message>
+        <source>Curl</source>
+        <translation>Kringel</translation>
+    </message>
+    <message>
+        <source>Twirl</source>
+        <translation>Wirbel</translation>
+    </message>
+    <message>
+        <source>Thickness</source>
+        <translation>Dicke</translation>
+    </message>
+</context>
+<context>
+    <name>TFurStrokeStyle</name>
+    <message>
+        <source>Herringbone</source>
+        <translation>Fischgrätenstich</translation>
+    </message>
+    <message>
+        <source>Angle</source>
+        <translation>Winkel</translation>
+    </message>
+    <message>
+        <source>Size</source>
+        <translation>Größe</translation>
+    </message>
+</context>
+<context>
+    <name>TGraphicPenStrokeStyle</name>
+    <message>
+        <source>Dashes</source>
+        <translation>Bindestriche</translation>
+    </message>
+    <message>
+        <source>Density</source>
+        <translation>Dichte</translation>
+    </message>
+</context>
+<context>
+    <name>TLinGradFillStyle</name>
+    <message>
+        <source>Linear Gradient</source>
+        <translation>Linearer Gradient</translation>
```

Comment:
Gradient is mostly translated with Verlauf in the context of image manipulation.